### PR TITLE
Add service name + namespace to grafana dashboard metrics

### LINKED ISSUE #24: Varnish operator grafana dashboard pulls in too many metrics
The Grafana dashboard that is deployed with the operator pulls in metrics from other pods in the cluster that are unrelated to varnish. See the `CPU usage` and `Memory usage` graphs in the dashboard. We should narrow the scope of the metrics that are used in the dashboard. Additionally, the `CPU usage` graph appears to use time when it should be using percentage.

### PR DESCRIPTION
**Pull request by kscharm**
Added the varnish operator service name and release namespace to filter each metric in the Grafana dashboard deployed with the operator. Also changed the format for CPU usage to percentage.

Resolves #24.

### CODE REVIEW

File: config/samples/varnishcluster.yaml [hunk 1]
```diff
@@ -12,8 +12,6 @@ spec:
     # path to image + tag
     image: ibmcom/varnish:0.27.1
     imagePullPolicy: Always
-    # restart policy used for Varnish containers
-    restartPolicy: Always
```
Comment (tomashibm) [inline config/samples/varnishcluster.yaml:16]: was there anything wrong with that config?
I don't mind commenting it if necessary, but let's have it here so users can see all available fields with comments in this sample.

**Review by tomashibm — CHANGES_REQUESTED** [review]

File: varnish-operator/templates/grafana-dashboard.yaml [hunk 2]
```diff
@@ -644,7 +644,7 @@ data:
         },
         "yaxes": [
         {
-          "format": "s",
+          "format": "percent",
```
Comment (tomashibm) [inline varnish-operator/templates/grafana-dashboard.yaml:647]: the metrics itself are returned in seconds (`process_cpu_seconds_total`). You sure grafana will convert automatically seconds to percents correctly?

File: config/samples/varnishcluster.yaml (hunk 1 above)
Comment (kscharm) [reply config/samples/varnishcluster.yaml:16]: I tried to apply the base VarnishCluster CRD and it returned an error saying this field did not exist.

Comment (tomashibm) [reply config/samples/varnishcluster.yaml:16]: ah, right. We removed it when we moved to statefulsets where `Always` is the only allowed value. Good catch!

File: varnish-operator/templates/grafana-dashboard.yaml (hunk 2 above)
Comment (kscharm) [reply varnish-operator/templates/grafana-dashboard.yaml:647]: Yea I'm not sure about this but time does not seem like the correct unit. Do you know of another metric we can use that returns CPU usage as a percentage?

Comment (tomashibm) [reply varnish-operator/templates/grafana-dashboard.yaml:647]: it's correct, just a different way - it tells you how much CPU time the process consumed. e.g. 1 second would be the process did computations for one second on one core.
I didn't find a separate percentage metric for that, but I think you can use this one. Just multiply the metric by 100.
e.g. `irate(process_cpu_seconds_total[1m]) * 100`.

Comment (allen-servedio) [reply varnish-operator/templates/grafana-dashboard.yaml:647]: @tomashibm - you can talk with Tym about this to see what we are using over in our systems.

**Review by tomashibm — APPROVED** [review]

**Review by cin — APPROVED** [review]
LGTM!

**Pull request merged**